docs/pages/access-controls/compliance-frameworks/fedramp.mdx

```diff
@@ -25,29 +27,43 @@ government agencies.
 | [IA-03 Device Identification and Authentication]((=fedramp.control_url=)IA-3) | Teleport requires valid x509 or SSH certificates issued by a Teleport Certificate Authority (CA) to establish a network connection for device-to-device network connection between Teleport components. |
 | [SC-12 Cryptographic Key Establish and Management]((=fedramp.control_url=)SC-12) | Teleport initializes cryptographic keys that act as a Certificate Authority (CA) to further issue x509 and SSH certificates. SSH and x509 user certificates that are issued are signed by the CA and are (by default) short-lived. SSH host certificates are also signed by the CA and rotated automatically (a manual force rotation can also be performed).<br/>Teleport Enterprise builds against a FIPS 140-2 compliant library (BoringCrypto) is available. <br/>In addition, when Teleport Enterprise is in FedRAMP/FIPS 140-2 mode, Teleport will only start and use FIPS 140-2 compliant cryptography. |
 
-## Setup
+## Download and install
 
-If your Teleport cluster is running on AWS, it must be running in a GovCloud
-region to support FIPS.
-
-Sign in to your [Teleport account](https://teleport.sh) to download a FIPS binary
-for Teleport Enterprise. Look for `Linux 64-bit (FedRAMP/FIPS)`. RPM and DEB
-packages are also available.
+You can follow the [Installation instructions](../../installation.mdx#installation-instructions) for 
+Teleport Enterprise edition to download and install the appropriate FIPS-compliant binaries for 
+your operating environment and package manager or from compressed archive (tarball).
 
-After downloading the binary tarball, run:
+For example, you can download and install from the compressed archive by running the following commands:
 
 ```code
-$ tar -xzf teleport-ent-v(=teleport.version=)-linux-amd64-fips-bin.tar.gz
+$ curl https://get.gravitational.com/teleport-ent-(= teleport.version =)-linux-<Var name="$SYSTEM_ARCH"/>-fips-bin.tar.gz.sha256
+<checksum> <filename>
+$ curl -O https://cdn.teleport.dev/teleport-ent-(= teleport.version =)-linux-<Var name="$SYSTEM_ARCH"/>-fips-bin.tar.gz
+
+# Verify that the checksums match
+$ shasum -a 256 teleport-ent-(= teleport.version =)-linux-<Var name="$SYSTEM_ARCH"/>-fips-bin.tar.gz
+
+$ tar -xvf teleport-ent-(= teleport.version =)-linux-<Var name="$SYSTEM_ARCH"/>-fips-bin.tar.gz
 $ cd teleport-ent
 $ sudo ./install
-# This will copy Teleport Enterprise to /usr/local/bin.
 ```
 
-## Configuration
+After you download and install, all of the Teleport Enterprise binaries are 
+installed in the `/usr/local/bin` directory. You can verify you have FIPS-compliant
+binaries installed by running the `teleport version` command and verifying that 
+the `X:boringcrypto` library is listed. For example:
+
+```code
+$ teleport version
+Teleport Enterprise v13.3.9 git:api/v13.3.9-0-gd1a97d9 go1.20.8 X:boringcrypto
+```
+
+If your Teleport cluster runs on AWS, the cluster must run in a GovCloud
```

Comment:
This isn't correct. AWS US East/West can be used for FedRAMP authorized services up to Moderate impact level.